libcudacxx/include/cuda/std/detail/libcxx/include/__config

```diff
@@ -37,6 +37,18 @@
 #  define _LIBCUDACXX_COMPILER_CLANG_CUDA
 #endif
 
+// Some convenience macros to filter nvcc versions
+#if defined(_LIBCUDACXX_COMPILER_NVCC) && _LIBCUDACXX_CUDACC_VER < 1102000
+#define _LIBCUDACXX_COMPILER_NVCC_BELOW_11_2
```

Comment:
What's the story with why we're manually giving a name to this expression as opposed to just using the `< N` logic in the preprocessor guard where these are being used? Is the logical conclusion of this that we should add a definition like this for every possible nvcc version we support?

Reply:
It is much tidier like that. Also we have some preprocessor directives, where we use `||` which made it a tad easier to screw up

Reply:
And no, we should only do this when it is frequent. For now we only ever really use 11_3 and 11_8 which are rather frequent